shell.nix

```diff
@@ -6,20 +6,20 @@ pkgs.mkShell {
   env.LC_ALL = "C.UTF-8";
 
   packages = [
-    pkgs.gcc
+    pkgs.SDL2
     pkgs.cmake
-    pkgs.zlib
-    pkgs.zstd
+    pkgs.curl
+    pkgs.freetype
+    pkgs.gcc
+    pkgs.gettext
+    pkgs.libGL
     pkgs.libjpeg
+    pkgs.libogg
     pkgs.libpng
-    pkgs.libGL
-    pkgs.SDL2
-    pkgs.openal
-    pkgs.curl
     pkgs.libvorbis
-    pkgs.libogg
-    pkgs.gettext
-    pkgs.freetype
+    pkgs.openal
     pkgs.sqlite
+    pkgs.zlib
+    pkgs.zstd
```

Comment:
what changed here and why?

Reply:
reordered the list to be alphabetical for easy manual scanning and comparison with vcpkg.json

Reply:
Undoing this change as I'm no longer using NixOS and I don't want to push untested changes